examples/playbook/sample.tlspc.yaml

```diff
@@ -0,0 +1,30 @@
+config:
+  connection:
+    type: vaas
+    credentials:
+      apikey: "XxXXxXxx-Xxxx-XXxX-XxXx-xXxxXXxxXXxX"
+certificates:
+  - name: myCertificate # Task Identifier, no relevance in tool run
+    request:
+      csrOrigin: service
+      keyPassword: "newPassword!"
+      subject:
+        # Templating needs to go between single quotes to avoid issues when refreshing tokens and saving back
+        commonName: '{{ Hostname | ToLower -}}.{{- Env "USERDNSDOMAIN" | ToLower }}'
+        country: US
+        locality: Salt Lake City
+        province: Utah
+        organization: Venafi Inc
+        orgUnits:
+          - engineering
+          - marketing
+      zone: "Open Source\\vcert"
+    installations:
+      - type: PEM
+        afterInstallAction: "echo Success!!!"
+        # Notice this is a folder path. Certificate, Chain and Private Key files will be created here
+        location: "/path/to/my/certificate/folder"
+        pemCertFilename: "cert.cer"
+        pemChainFilename: "chain.cer"
+        pemKeyFilename: "key.pem"
+    renewBefore: 31d
```

Comment:
Let's add one newline here. Otherwise, Github complains with:
![image](https://github.com/Venafi/vcert/assets/77068025/6e72df12-ed66-410a-b968-530c6a010537)